### ISSUE TITLE
Run two A/B tests with Optimizely

### ISSUE THREAD

**Issue by peapumpkin** [spring-giant/mgt656-spring-2015-project#49]
2 things we need to test with A/B:
1) RSVP on homepage: if making the event links on the home page include the word "RSVP" would increase click-through rate
2) Donate vs support link:  if they should ask people to "donate" or "support" an event in the the donation link text to increase conversions
Write up analytics results in report.

**Comment by peapumpkin** [comment]
Jeff and Lea co-attack this user story.

**Comment by peapumpkin** [comment]
I've set up the Optimizely account (email: lea.lupkin@yale.edu, password: SpringGiant656). Now need to set up the 2 experiments.

**Comment by meghaghildyal** [comment]
Are we putting this in sprint-5 report? I am asking because I am writing the report right now and wont have time tomorrow to do it.

**Comment by peapumpkin** [comment]
Yes say that we set up the account and are going to be doing the 2 experiments this upcoming sprint.

**Comment by peapumpkin** [comment]
1) Sent the changes for the RSVP a/b test for Merge.  Experiment is running google analytics, so hopefully one my changes are merged it will start tracking.  We should be able to try out the experiment ourselves if we visit the main website url. 2) I haven't figure out the donate vs. support a/b test yet, how to set up the code in the cloud9 and then how to set up the goal in experiment on google analytics. Sent long email to Jeff asking for his help and updating him on what I have so far. 3) I've spent 8 hours+ on this already.

**Comment by meghaghildyal** [comment]
merged an hour ago
